airflow/migrations/versions/92c57b58940d_add_fab_tables.py

```diff
@@ -0,0 +1,173 @@
+            sa.Column('last_name', sa.String(length=64), nullable=False),
+            sa.Column('username', sa.String(length=64), nullable=False),
+            sa.Column('password', sa.String(length=256), nullable=True),
+            sa.Column('active', sa.BOOLEAN(), nullable=True),
+            sa.Column('email', sa.String(length=64), nullable=False),
+            sa.Column('last_login', sa.DATETIME(), nullable=True),
+            sa.Column('login_count', sa.INTEGER(), nullable=True),
+            sa.Column('fail_login_count', sa.INTEGER(), nullable=True),
+            sa.Column('created_on', sa.DATETIME(), nullable=True),
+            sa.Column('changed_on', sa.DATETIME(), nullable=True),
+            sa.Column('created_by_fk', sa.INTEGER(), nullable=True),
+            sa.Column('changed_by_fk', sa.INTEGER(), nullable=True),
+            sa.ForeignKeyConstraint(['changed_by_fk'], ['ab_user.id']),
+            sa.ForeignKeyConstraint(['created_by_fk'], ['ab_user.id']),
+            sa.PrimaryKeyConstraint('id'),
+            sa.UniqueConstraint('email'),
+            sa.UniqueConstraint('username'),
+        )
+
+    if "ab_user_role" not in tables:
+        op.create_table(
+            'ab_user_role',
+            sa.Column('id', sa.INTEGER(), nullable=False, primary_key=True),
+            sa.Column('user_id', sa.INTEGER(), nullable=True),
+            sa.Column('role_id', sa.INTEGER(), nullable=True),
+            sa.ForeignKeyConstraint(
+                ['role_id'],
+                ['ab_role.id'],
+            ),
+            sa.ForeignKeyConstraint(
+                ['user_id'],
+                ['ab_user.id'],
+            ),
+            sa.PrimaryKeyConstraint('id'),
+            sa.UniqueConstraint('user_id', 'role_id'),
+        )
+
+    if "ab_register_user" not in tables:
+        op.create_table(
+            'ab_register_user',
+            sa.Column('id', sa.INTEGER(), nullable=False, primary_key=True),
+            sa.Column('first_name', sa.String(length=64), nullable=False),
+            sa.Column('last_name', sa.String(length=64), nullable=False),
+            sa.Column('username', sa.String(length=64), nullable=False),
+            sa.Column('password', sa.String(length=256), nullable=True),
+            sa.Column('email', sa.String(length=64), nullable=False),
+            sa.Column('registration_date', sa.DATETIME(), nullable=True),
+            sa.Column('registration_hash', sa.String(length=256), nullable=True),
+            sa.PrimaryKeyConstraint('id'),
+            sa.UniqueConstraint('username'),
+        )
+
+    # Increase length of ab_view_menu.name column
+    with op.batch_alter_table('ab_view_menu') as batch_op:
+        batch_op.alter_column('name', type_=sa.String(length=250), nullable=False)
+
+
+def downgrade():
+    """Drop FAB Tables"""
+    conn = op.get_bind()
+    inspector = Inspector.from_engine(conn)
+    fab_tables = [table for table in inspector.get_table_names() if table.startswith("ab_")]
```

Comment:
I'd feel more comfortable explicitly listing the tables we add above

Reply:
Done in https://github.com/apache/airflow/pull/12352/commits/c4d4c78951228b0ec7a598b151fa0142e6191a3e